src/system_window.cpp

```diff
@@ -658,7 +658,20 @@ void SystemWindow::GetInput( SystemEvents& out_events )
 			out_events.back().event.mouse_move.dx= event.motion.xrel;
 			out_events.back().event.mouse_move.dy= event.motion.yrel;
 			break;
-
+		case SDL_MOUSEWHEEL:
+			out_events.emplace_back();
+			out_events.back().type=SystemEvent::Type::Wheel;
+			out_events.back().event.wheel.dx = event.wheel.direction == SDL_MOUSEWHEEL_FLIPPED ? -event.wheel.preciseX : event.wheel.preciseX;
```

Comment:
`error: no member named 'preciseY' in 'SDL_MouseWheelEvent'`
I suppose this member was added in newer versions of SDL2 library. I think we should still support old versions (version, which was used in initial releases).

Reply:
Floating point wheel has been introduced in SDL-2.0.18 and wheel direction in SDL-2.0.4.
For PanzerChasm 1-axis integer only mouse wheel support seems to be sufficient.
Should I reduce the changes to it and use only `wheel.delta`?

Reply:
`wheel.delta` will be enough.